### ISSUE TITLE
[EN][Quest][Deepholm] The Twilight Overlook (minor)

### ISSUE THREAD

**Issue by tdotzlod** [Atlantiss/BugTracker#4621]
This quest is a part of chain leading to First Fragment.
http://www.wowhead.com/quest=27061/the-twilight-overlook
Player supposed to turn it to Stormcaller Mylra (http://www.wowhead.com/npc=44010/stormcaller-mylra).

Right now this quest getting turned into air. Player complete it on spot(without going to dedicated npc)which cause majority of people getting lost and being unable to start therazane's chain.

**Comment by Kobiano** [comment]
Yes, I completed this quest on spot and then I was left with no quests. I've flown round every inch of the map and couldn't find any quest givers. Now I'm stuck and cant start the Therazane chain.

**Comment by Kobiano** [comment]
Yes, I completed this quest and now I don't have any other quests. I've flown around every inch of the map and couldn't find any quest givers. Now I'm stuck and can't start the Therazane chain.
